network-api/networkapi/templates/buyersguide/fragments/consumer_creepometer_page_2023/outro.html

```diff
@@ -0,0 +1,28 @@
+<div class="{{ wrapper_class }} tw-bg-gradient-to-b tw-from-white tw-to-purple-05">
+  <div class="tw-container">
+    <div class="tw-grid tw-grid-cols-1 large:tw-grid-cols-3 tw-gap-x-6 tw-gap-6 large:tw-gap-y-0 tw-mb-[120px]">
+      <div class="tw-col-span-1 tw-rounded-2xl tw-bg-white">
+        <div class="tw-h-[200px] large:tw-h-full tw-w-full tw-rounded-2xl tw-text-center tw-bg-no-repeat tw-bg-[center_-8px] large:tw-bg-center tw-bg-[length:auto_300px] medium:tw-bg-cover tw-bg-[url('../_images/buyers-guide/consumer-creepometer/our-approach.png')]"></div>
```

Comment:
@jhonatan-lopes thanks for bringing this up. Most of the stylings on this one-off page are custom (and very custom). Since they are only applicable to this particular page, I thought it might not be needed to add them to Tailwind config.

Reply:
Agreed that we should have a discussion with our designer peers for this for future projects.

Reply:
That's exactly the use case for custom classes, thanks for clarifying!